### ISSUE TITLE
Comparison with other SSL baseline

### ISSUE THREAD

**Issue by liuyuchen6666** [zhengzangw/PCS-FUDA#6]
Thanks for your great job. I wonder have you compared with SSL baseline, e.g., moco, simclr? I try moco and simclr with the model initialized by ImageNet pre-trained. After training several epochs, it seems the kNN accuracy of the trained model is even worse than the pre-trained model. Do you have any insight into this situation? Thanks a lot.

**Comment by zhengzangw** [comment]
We do not try the moco and simclr method, but we have tried InstDiscrimination, which is also an instance-level SSL task. We find its performance is worse than the cluster-based method. We think the reason is that when there are a limited number of samples, cluster-based methods are more robust.

**Comment by liuyuchen6666** [comment]
Thanks for your reply. I wonder the hyperparamter for InstDiscrimination, i.e., batch size, learning rate, optimizer. Since it can also improve the pre-trained model by 8% accuracy. Thanks a lot.
![1650453803(1)](https://user-images.githubusercontent.com/55131290/164220188-607840c1-64fc-4573-be1d-49ba94bd455e.png)

**Comment by zhengzangw** [comment]
Sorry, the config was on the server and is not available at present. But the batch size, learning rate, and optimizer are almost the same as our protoNCE method. You can take advantage of the `_compute_ID_loss` function in `pcs/models/ssda` to replicate the result.
